lookout/style/format/features.py

```diff
@@ -82,7 +85,16 @@ def from_node(node: bblfsh.Node, file: str, path: str) -> Iterable["VirtualNode"
                               Position(*[f[1] for f in node.end_position.ListFields()]),
                               node=node, path=path)
             return
-        start_offset = outer_token.find(node.token)
+        token = node.token
+        if node.internal_type in interpreted_types:
+            token, outer_token = interpreted_types[node.internal_type](token, outer_token)
```

Comment:
Looks like we never need `token` in the function call - it is interpreted so we never trust it - and this argument can be removed